### ISSUE TITLE
Gnome 3.24: gtk ignores theme settings set by Cinnamon

### ISSUE THREAD

**Issue by vladimir-didenko** [linuxmint/Cinnamon#6375]
After upgrading to Gnome 3.24 I noticed that gtk doesn't respect theme settings set by cinnamon-settings -
 org.cinnamon.desktop.interface.\*. Gnome settings org.gnome.desktop.interface.\* are used instead.

**Comment by pss959** [comment]
I am having the same issue. Font, control theme, and icon theme settings made through cinnamon have no effect.

**Comment by irwtdvoys** [comment]
Logging into Gnome Shell and changing the settings with gnome-tweak-tool fixed it for me, something odd going on with 3.24 though

**Comment by nick-s-b** [comment]
After upgrading my Arch install today, Gnome completely messed up Cionnamon for me. You can't change any theme settings anymore.

**Comment by leigh123linux** [comment]
I believe gsd plugins are being autostarted which seem to block the csd plugins.

https://git.gnome.org/browse/gnome-settings-daemon/commit/?id=6d01a434ca2faa88acb2aed1224d23cea329f1f8

Fedora cinnamon is working ok, I applied this closed pr

https://github.com/linuxmint/cinnamon-session/pull/85

**Comment by leigh123linux** [comment]
To test, run this as non-root

```
gsettings set org.cinnamon.SessionManager autostart-blacklist "['gnome-settings-daemon', 'gnome-fallback-mount-helper', 'gnome-screensaver', 'mate-screensaver', 'mate-keyring-daemon', 'indicator-session', 'gnome-initial-setup-copy-worker', 'gnome-initial-setup-first-login', 'gnome-welcome-tour', 'xscreensaver-autostart', 'nautilus-autostart', 'caja', 'xfce4-power-manager']"
[leigh@localhost ~]$ gsettings set org.cinnamon.SessionManager autostart-blacklist "['gnome-settings-daemon', 'gnome-fallback-mount-helper', 'gnome-screensaver', 'mate-screensaver', 'mate-keyring-daemon', 'indicator-session', 'gnome-initial-setup-copy-worker', 'gnome-initial-setup-first-login', 'gnome-welcome-tour', 'xscreensaver-autostart', 'nautilus-autostart', 'caja', 'xfce4-power-manager', 'org.gnome.SettingsDaemon.A11yKeyboard', 'org.gnome.SettingsDaemon.A11ySettings', 'org.gnome.SettingsDaemon.Clipboard', 'org.gnome.SettingsDaemon.Color', 'org.gnome.SettingsDaemon.Datetime', 'org.gnome.SettingsDaemon.Housekeeping', 'org.gnome.SettingsDaemon.Keyboard', 'org.gnome.SettingsDaemon.MediaKeys', 'org.gnome.SettingsDaemon.Mouse', 'org.gnome.SettingsDaemon.Orientation', 'org.gnome.SettingsDaemon.Power', 'org.gnome.SettingsDaemon.PrintNotifications', 'org.gnome.SettingsDaemon.Rfkill', 'org.gnome.SettingsDaemon.ScreensaverProxy', 'org.gnome.SettingsDaemon.Sharing', 'org.gnome.SettingsDaemon.Smartcard', 'org.gnome.SettingsDaemon.Sound', 'org.gnome.SettingsDaemon.Wacom', 'org.gnome.SettingsDaemon.XRANDR', 'org.gnome.SettingsDaemon.XSettings']"
```

**Comment by leigh123linux** [comment]
Archlinux applied my fix yesterday so it should be fixed.

https://git.archlinux.org/svntogit/community.git/commit/trunk?h=packages/cinnamon-session&id=c9aa1a9eeb6246e420fdd9e0dcf94f8a8289580d

**Comment by nick-s-b** [comment]
Thanks so much @leigh123linux !! much appreciated!

**Comment by pcov** [comment]
Ive installed this desktop on Ubuntu 16.04, and it works fantastically apart from a small bug. I have changed my desktop wallpaper and have tried to send a few desktop shortcuts on it and it doesnt show them, but if you look at the desktop in \files then it does show them to be there. also occasionaly the desktop background reverts back to the wallpaper set under ubuntu's desktop manager, and it shows the missing shortcuts that I had installed. I'm a bit of a Linux Noob, but am liking it and learning as quick as I can. Hasanyone else experienced these bugs, any help would be greatly appreciated.
